src/main/java/mops/testenviroment/DatabaseInit.java

```diff
@@ -7,22 +7,29 @@
 import mops.model.classes.Certificate;
 import mops.model.classes.Distribution;
 import mops.model.classes.Evaluation;
+import mops.model.classes.Module;
+import mops.model.classes.Priority;
+import mops.model.classes.Role;
 import mops.repositories.ApplicantRepository;
 import mops.repositories.ApplicationRepository;
 import mops.repositories.DistributionRepository;
 import mops.repositories.EvaluationRepository;
+import mops.repositories.ModuleRepository;
 import org.springframework.boot.web.servlet.ServletContextInitializer;
 import org.springframework.stereotype.Component;
 
 import javax.servlet.ServletContext;
 import java.text.SimpleDateFormat;
+import java.time.Instant;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
+@SuppressWarnings("checkstyle:MagicNumber")
```

Comment:
wenn ENTRYNUMBER final ist, braucht man dann noch ein SuppressWarning?

Reply:
Das bezieht sich auf alle Magicnumbers in der Init (Stundenanzalh/ Random-Wertebereiche/usw.)